plugin/loader/preload.go

```diff
@@ -6,6 +6,7 @@ import (
 	pluginflatfs "github.com/ipfs/go-ipfs/plugin/plugins/flatfs"
 	pluginipldgit "github.com/ipfs/go-ipfs/plugin/plugins/git"
 	pluginlevelds "github.com/ipfs/go-ipfs/plugin/plugins/levelds"
+	pluginrados "github.com/ipfs/go-ipfs/plugin/plugins/rados"
```

Comment:
Shouldn't be preloaded by default as
* It's not officially supported / well tested (at least yet)
* go-ceph uses cgo

Reply:
Ok, I will remove it.

Reply:
DONE